examples/account/revoke_vc.rs

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This example shows how to revoke a verifiable credential.
+//!
+//! The Verifiable Credential is revoked by actually removing a verification method (public key)
+//! from the DID Document of the Issuer.
+//! As such, the Verifiable Credential can no longer be validated.
+//! This would invalidate every Verifiable Credential signed with the same public key, therefore the
+//! issuer would have to sign every VC with a different key.
+//!
+//! Note: This example uses in-memory storage for simplicity.
+//! See `create_did` example to configure Stronghold storage.
+//!
+//! cargo run --example revoke_vc
+
+use identity::account::Account;
+use identity::account::IdentitySetup;
+use identity::account::MethodContent;
+use identity::core::json;
+use identity::core::FromJson;
+
+use identity::core::Url;
+use identity::credential::Credential;
+use identity::credential::CredentialBuilder;
+use identity::credential::Subject;
+use identity::crypto::ProofOptions;
+
+use identity::did::DID;
+
+use identity::iota::CredentialValidationOptions;
+use identity::iota::CredentialValidator;
+
+use identity::iota::ResolvedIotaDocument;
+
+use identity::iota::Resolver;
+
+use identity::account::Result;
+mod create_did;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+  // ===========================================================================
+  // Create a Verifiable Credential.
+  // ===========================================================================
+
+  // Create an identity for the issuer.
+  let mut issuer: Account = Account::builder().create_identity(IdentitySetup::default()).await?;
+
+  // Add verification method to the issuer.
+  issuer
+    .update_identity()
+    .create_method()
+    .content(MethodContent::GenerateEd25519)
+    .fragment("key-1")
+    .apply()
+    .await?;
+
+  // Create a credential subject indicating the degree earned by Alice.
+  let subject: Subject = Subject::from_json_value(json!({
+    "id": "did:iota:B8DucnzULJ9E8cmaReYoePU2b7UKE9WKxyEVov8tQA7H",
+    "name": "Alice",
+    "degree": "Bachelor of Science and Arts",
+    "GPA": "4.0",
+  }))?;
+
+  // Build credential using subject above and issuer.
+  let mut credential: Credential = CredentialBuilder::default()
+    .id(Url::parse("https://example.edu/credentials/3732")?)
+    .issuer(Url::parse(issuer.did().as_str())?)
+    .type_("UniversityDegreeCredential")
+    .subject(subject)
+    .build()?;
+
+  // Sign the Credential with the issuer's verification method.
+  issuer.sign("#key-1", &mut credential, ProofOptions::default()).await?;
+
+  // ===========================================================================
+  // Revoke the Verifiable Credential.
+  // ===========================================================================
+
+  // Remove the public key that signed the VC from the issuer's DID document
+  // - effectively revoking the VC as it will no longer be able to verified.
```

Comment:
```suggestion
  // Remove the public key that signed the VC from the issuer's DID document.
  // This effectively revokes the VC as it will no longer be able to be verified.
```